src/components/Auth/SignUpForm.schema.js

```diff
@@ -0,0 +1,33 @@
+import Joi from '@hapi/joi';
+import { createValidationResolver } from '../form';
+
+const schema = Joi.object({
+  firstName: Joi.string()
+    .allow('')
+    .trim(),
+  lastName: Joi.string()
+    .allow('')
+    .trim(),
+  username: Joi.string()
+    .alphanum()
+    .min(3)
+    .max(30)
```

Comment:
are these numbers coming from a requirement?

Reply:
There weren't explicit requirements set in the issue (my bad) -- but 3 - 30 seems reasonable to me. 😅